README.md

```diff
@@ -33,16 +33,16 @@ definitions are axiom-free.
 - [Installation](#installation)
 - [HoTT Variant](#hott-variant)
 
-## Live demo
-
-Try it now in your browser with [JSCoq](http://mattam82.github.io/Coq-Equations/assets/jsexamples/equations_intro.html)!
-
 ## Documentation
 
+- You can discover and learn Equations through [interactive tutorials](https://coq.inria.fr/platform-docs/) 
+  directly in your browser, or locally with Equations installed.
+
 - The [reference manual](http://github.com/mattam82/Coq-Equations/raw/main/doc/equations.pdf)
-  provides an introduction and a summary of the commands and options.
-  This introduction can also be followed interactively with Equations installed:
+  provides a summary of the commands and options, and a brief introduction.
+  This introduction can also be followed interactively with Equations installed,
   [equations_intro.v](http://github.com/mattam82/Coq-Equations/raw/main/doc/equations_intro.v)
+  or directly [lived](http://mattam82.github.io/Coq-Equations/assets/jsexamples/equations_intro.html).
```

Comment:
"lived" sounds weird. I'm not sure you can use it that way and I wouldn't understand.

Reply:
right, what about 
```suggestion
  or [interactively in your browser](http://mattam82.github.io/Coq-Equations/assets/jsexamples/equations_intro.html).
```

Reply:
Yes, much better.